docs/use_cases/model_applications/clouds/GridStat_fcstGFS_obsGFS_cloudFracLayer.py

```diff
@@ -0,0 +1,329 @@
+"""
+Grid-Stat, MODE, Stat-Analysis, UserScript, Gen-Vx-Mask: GFS Cloud Statistics by Type
+=====================================================================================
+
+model_applications/clouds/GridStat_fcstGFS_obsGFS_cloudFracLayer.conf
+
+"""
+##############################################################################
+# .. contents::
+#   :depth: 1
+#   :local:
+#   :backlinks: none
+
+##############################################################################
+# Scientific Objective
+# --------------------
+#
+# This use case evaluates clouds classified into low, mid, and high levels.  The
+# evaluation covers multiple types of statistics including contingency table
+# statistics, neighborhood statistics, distance maps, and an object based evaluation.
+# Additionally, plots are created to demonstrate how to reformat and visualize GSS
+# CSI and frequency bias from the command line.
+
+##############################################################################
+# Version Added
+# -------------
+#
+# METplus version 6.1
+
+##############################################################################
+# Datasets
+# --------
+#
+# **Forecast**: GFS 0.25 degree model
+#
+# **Observation**: GFS Analysis
+#
+# **Climatology:** None
+#
+# **Location:** All of the input data required for this use case can be 
+# found in a sample data tarball. Each use case category will have 
+# one or more sample data tarballs. It is only necessary to download 
+# the tarball with the use case’s dataset and not the entire collection 
+# of sample data. Click here to access the METplus releases page and download sample data 
+# for the appropriate release: https://github.com/dtcenter/METplus/releases
+# This tarball should be unpacked into the directory that you will 
+# set the value of INPUT_BASE. See :ref:`running-metplus` section for more information.
+
+##############################################################################
+# METplus Components
+# ------------------
+#
+# This use case calls GenVxMask once, GridStat once, MODE three times, StatAnalysis
```

Comment:
Do you say "GridStat once" here to reflect GHA? I think if the user runs it MODE and GridStat both run the same number of times, is that correct? If so we may want to just say that here rather than write about what GHA does.

Reply:
It's more "GridStat once" to reflect the number of times it's called in the process list.

Reply:
OK gotcha- maybe an easy follow-up question then. How come MODE has three sections for low/mid/high cloud but GridStat doesn't need that? What specific to MODE is happening for each of those variables that can't be controlled via MODE_FCST_VAR<n>_NAME, etc.? Is it just `MODE_OUTPUT_PREFIX` that's changing?

Reply:
OK great! I may add some minor wording to the documentation to this effect for clarity, this makes perfect sense but wasn't immediately obvious to me. Thanks!

Reply:
Changes are in 992a6924cfe4cf701c0a96ae58ee9a8908779890